apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-extra-details/user-job/user-job.component.html

```diff
@@ -146,7 +146,7 @@
         </label>
         <label>
           Responsibilities & Perks
-          <textarea formControlName="description" fullWidth nbInput rows="3" type="text"></textarea>
```

Comment:
Why we remove row=3

Reply:
Because earlier we need to show top 3 rows and rest rows with scroll and after resize rows="3" has no effect

Reply:
![3 rows autoresize](https://user-images.githubusercontent.com/86765602/217735554-08e2462b-fab9-4424-9fb9-da2390374fcf.png)